### DISCUSSION TITLE
Unable To Get Secure Connection

### DISCUSSION THREAD

**Discussion by maxycatz** [jcorporation/myMPD#1272] [Q&A] [answered]
My previous install of myMPD didn't present any problems with certificates; it was transparent and the install as Chromium App just happened.  
I'm on a new install of RPI OS and have tried both installing from the repository and compiling myMPD v15 and I seem unable to get a secure connection within localhost.
I've tried going through what is in the documentation, but it's not exactly a step by step for someone with no previous experience of certificates and what needs to be trusted and import/placed where.
What am I missing?  Everything is most recent versions.

**Comment by jcorporation** [comment, marked as answer]
Download the generated myMPD ca certificate, it is linked in the About dialog. This certificate must be imported in the certificate trust store of your browser. If your browser uses the system trust store you must it import there. I can not provide a step by step guide, it is different for each browser /os.

I hope this helps.
